tests/test.mjs

```diff
@@ -355,7 +355,8 @@ if (process.versions.node.split('.')[0] >= 14) {
 	});
 }
 
-test('ADDRCONFIG hint', async t => {
+// FIXME: this shouldn't be serial
+test.serial('ADDRCONFIG hint', async t => {
```

Comment:
I guess `quibble` fails to update the imports somehow? I'm fine with `serial` for now...